resources/qml/dialogs/RoomMembers.qml

```diff
@@ -162,6 +162,10 @@ ApplicationWindow {
                         }
 
                         PowerlevelIndicator {
+                            Layout.preferredWidth: 32
```

Comment:
I know you didn't want to change the design, but this really shouldn't be 2x as big as the other one. If we are spelling out a size for this now, can we make both of them 2x the line height?

Reply:
Although I guess that won't work for your own profile?

Reply:
Made them both `fontMetrics.lineSpacing * 2`.